gmt/clib/utils.py

```diff
@@ -115,18 +120,22 @@ def _series_to_array(vector):
 
     >>> import pandas as pd
     >>> x_series = pandas.Series(data=[1, 2, 3, 4])
-    >>> x_array = _series_to_array(x_series)
+    >>> x_array = _as_array(x_series)
     >>> type(x_array)
     <class 'numpy.ndarray'>
     >>> x_array
     array([1, 2, 3, 4])
     >>> import numpy as np
-    >>> type(_series_to_array(np.array([5, 6, 7])))
+    >>> type(_as_array(np.array([5, 6, 7])))
+    <class 'numpy.ndarray'>
+    >>> type(_as_array([3, 4, 5]))
     <class 'numpy.ndarray'>
 
     """
     if isinstance(vector, pandas.Series):
         return vector.as_matrix()
+    elif isinstance(vector, list):
+        return np.array(vector)
```

Comment:
@seisman just found out that you can use [numpy.asarray](https://docs.scipy.org/doc/numpy/reference/generated/numpy.asarray.html) to do the conversion. In this case, even if the object is already an array the memory is not copied. So you can replace the isinstance check with:

```python
return np.asarray(vector)
```

This will also handle tuples, so we get an added bonus.